### PR TITLE
Small fixes in response and error handling

### PR DESCRIPTION
**Pull request by adonese**
I fixed errors in json handling + getKey logic

### CODE REVIEW

**Comment by adonese** [comment]
I know it works, tested it and performed a payment as well. But i honestly have no idea why it works! Would you please author and review this change. 

It took me a while to merge my changes with yours; really thanks for jetbrains for making this change 1000x harder than it should be 👎

File: lib/src/main/java/com/solus/sdk/noebscall/NOEBSClient.java [hunk 1]
```diff
@@ -127,9 +150,21 @@ private String getKey() {
...
                 errorBaseResponse.setResponse(errorResponse);
                 return errorBaseResponse;
+            }else if (response.code() == 502) { // for ebs error messages
+                String responseBodey = response.body().string();
+
+                JsonObject jsonObject = gson.fromJson(responseBodey, JsonObject.class);
+
+                logger.info("gson response " + jsonObject.get("details").toString());
+                logger.info("ebs_error_502 is: " + responseBodey);
+                ErrorResponse errorResponse = mapper.readValue(jsonObject.get("details").toString(), ErrorResponse.class);
```
Comment (fakhrisati) [inline lib/src/main/java/com/solus/sdk/noebscall/NOEBSClient.java]: **pass responsebody to readvalue func  instead of jsonObject**

Comment (adonese) [reply lib/src/main/java/com/solus/sdk/noebscall/NOEBSClient.java]: On it boss

**Review by fakhrisati — APPROVED** [review]

**Comment by fakhrisati** [comment]
Adding adonese fixes in response and error handling

**Pull request merged**